lib/sparkperf/build.py

```diff
@@ -60,7 +60,8 @@ def make_spark_distribution(commit_id, target_dir, spark_git_repo, merge_commit_
         # running PySpark on YARN or when running on Java 6.  Since we'll be building and running
         # Spark on the same machines and using standalone mode, it should be safe to
         # disable this warning:
-        run_cmd("./make-distribution.sh --skip-java-test")
+        # The `-T 1C` enables parallel Maven builds with 1 thread per CPU core
+        run_cmd("./make-distribution.sh --skip-java-test -T 1C")
```

Comment:
Yeah, I've removed it in the copy running on my cluster and am testing to see if that fixes things.